build.xml

```diff
@@ -70,7 +69,7 @@
         </exec>
         <exec executable="${path.ecs.executable}" logoutput="true" passthru="true" checkreturn="true">
             <arg value="check" />
-            <arg value="--config=${path.ecs.configuration}"/>
+            <arg value="--clear-cache" />
```

Comment:
Clearing the cache with every execution of `standards-packages` might be a bit painful as there is no `standards-packages-diff` variant and it is called during `build-demo-dev`.

Please reconsider this behavior.

Reply:
clearing of cache was removed